### DISCUSSION TITLE
Reranking in Milvus Lite

### DISCUSSION THREAD

**Discussion by Pratyush2758** [milvus-io/milvus#34408] [Q&A and General discussion]
Does Milvus Lite use the reranking strategy or the reranking strategy can be used with the complete Milvus version only?

**Comment by codingjaguar** [comment]
Hi @Pratyush2758 Milvus Lite is going to support hybrid search API (https://milvus.io/docs/multi-vector-search.md), it should be released within a week.

The hybrid search API supports RRF and weighted reranking. If you'd like to use reranker model for reranking, you can do that on the client side after receiving the topK results, e.g. using pymilvus[model] lib https://milvus.io/docs/rerankers-overview.md
